emu_sv/lindblad_operator.py

```diff
@@ -64,6 +64,7 @@ def __init__(
         self.pulser_linblads: list[torch.Tensor] = pulser_linblads
         self.device: torch.device = device
         self.complex = self.phis.any()
+        self.factored_shape = (2,) * self.nqubits
```

Comment:
wouldn't `(2,)*nqubits` trigger the limit on qubit shape ?

Reply:
this line of code should be deleted, it was for the old implementation, I'll remove it, thanks